src/lib/stores/event-view.ts

```diff
@@ -55,14 +46,11 @@ export const supportsReverseOrder = derived(
 );
 
 export const eventSortOrder: Readable<EventSortOrder> = derived(
-  [eventFilterSort, supportsReverseOrder, eventSortParam],
-  ([$eventFilterSort, $supportsReverseOrder, $eventSortParam]) => {
-    let sortOrder: EventSortOrder;
+  [eventFilterSort, supportsReverseOrder],
+  ([$eventFilterSort, $supportsReverseOrder]) => {
+    let sortOrder: EventSortOrder = 'ascending';
```

Comment:
Shouldn't we default to "descending" and does this totally drop support for the query param?

Reply:
No we should default to 'ascending' since not all versions support reverse order. If it does support reverse order, the eventFilterSort store is defaulted to 'descending' so first time users will still get reverse order if the version supports it. The sort change button updates the query param, this eventSortParam store was only used in this derived store (I'm not totally sure why we even had it in the first place, guessing a sveltekit issue)

Reply:
Yea, we definitely need to rethink our state management.